vendor/ember-cli-qunit/test-loader.js

```diff
@@ -1,18 +1,25 @@
 /* globals jQuery,QUnit */
 
 jQuery(document).ready(function() {
-  var TestLoaderModule = require('ember-cli/test-loader');
-  var TestLoader = TestLoaderModule['default'];
-  var addModuleIncludeMatcher = TestLoaderModule['addModuleIncludeMatcher'];
+  const TestLoaderModule = require('ember-cli/test-loader');
```

Comment:
1. I prefer the "liberal-let" strategy as coined in madhatted.com/2016/1/25/let-it-be
2. `vendor/` is not processed by babel, so these will not get transpiled.